### DISCUSSION TITLE
Upload to GitHub

### DISCUSSION THREAD

**Discussion by uJeFFF** [alangrainger/share-note#134] [Vote for upcoming features]
Hi, firstly I would like to thank you for the amazing job doing this plugin, it is really cool and works fine.

I would like to ask you if possible to make a connection in a way that I could, instead of upload for the plugin website, to my GitHub. This way, even if one day you do not continue to update or there is no more way to continue hosting the notes, we could upload it for our own GitHub and continue using the plugin anyway.

Thank you very much.

**Comment by berkdemir** [comment]
I don't think this is relevant for the plugin. You can use git to do that.

**Comment by alangrainger** [comment]
Yes, not relevant for this plugin. You can always spin up [your own backend server](https://github.com/note-sx/server) in the event I stop providing one (most likely would only happen if I die or Obsidian no longer exists).
